Project.toml

```diff
@@ -11,7 +11,7 @@ LLVM = "929cbde3-209d-540e-8aea-75f648917ca0"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 
 [compat]
-CUDA = "2, 3"
+CUDA = "^3.5"
```

Comment:
Caret specifier is the default, so you don't need to specify it. Just `"3.5"` is ehough.